docs/Account/integration.mdx

```diff
@@ -44,57 +44,30 @@ pnpm add @biconomy/account @biconomy/bundler @biconomy/paymaster @biconomy/commo
 </TabItem>
 </Tabs>
 
-## Integration Example
+## Integration
+
+Now you can simply create smart accounts like this.
 
 ```typescript
-import {
-  BiconomySmartAccountV2,
-  DEFAULT_ENTRYPOINT_ADDRESS,
-} from "@biconomy/account";
-import {
-  ECDSAOwnershipValidationModule,
-  DEFAULT_ECDSA_OWNERSHIP_MODULE,
-} from "@biconomy/modules";
-import { ChainId } from "@biconomy/core-types";
-import { IBundler, Bundler } from "@biconomy/bundler";
-import { IPaymaster, BiconomyPaymaster } from "@biconomy/paymaster";
-import { Signer } from "ethers";
-
-// create instance of bundler
-const bundler: IBundler = new Bundler({
-  //https://dashboard.biconomy.io/ get bundler urls from your dashboard
-  bundlerUrl: "",
-  chainId: ChainId.POLYGON_MUMBAI,
-  entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
-});
-
-// create instance of paymaster
-const paymaster: IPaymaster = new BiconomyPaymaster({
-  //https://dashboard.biconomy.io/ get paymaster urls from your dashboard
-  paymasterUrl: "",
-});
-
-// instance of ownership module
-const ownerShipModule = await ECDSAOwnershipValidationModule.create({
-  signer: {} as Signer, // ethers signer object
-  moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE,
-});
-
-const biconomySmartAccount = await BiconomySmartAccountV2.create({
-  chainId: ChainId.POLYGON_MUMBAI, //or any chain of your choice
-  bundler: bundler, // instance of bundler
-  paymaster: paymaster, // instance of paymaster
-  entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS, //entry point address for chain
-  defaultValidationModule: ownerShipModule, // either ECDSA or Multi chain to start
-  activeValidationModule: ownerShipModule, // either ECDSA or Multi chain to start
-});
-const address = await biconomySmartAccount.getAccountAddress();
+  import { ethers } from "ethers";	
+  import { BiconomySmartAccountV2} from "@biconomy/account";
+  import { PaymasterMode } from "@biconomy/paymaster";
+
+  let provider = new ethers.providers.JsonRpcProvider("rpcUrl");
+  let signer = new ethers.Wallet("private key", provider);
+
+  const biconomySmartAccountConfig = {
+    signer: signer,
+    chainId: 80001
+    biconomyPaymasterApiKey: "",
+    bundlerUrl: ""
+  };
+
+  const biconomySmartAccount = await BiconomySmartAccountV2.create(
+    biconomySmartAccountConfig
+  );
 ```
-
-:::info
-
-Make sure to call both create methods from an async function!
-
-:::
-
-Regardless of what frontend framework you may use these steps for creating a smart account will remain the same. See our [tutorials](/category/tutorials) for in depth integrations of the smart account.
+This smart account uses the following default values:
+- ECDSA Validation Module
+- default entry point address for the chainId
+- paymaster mode is set to SPONSORED
```

Comment:
Is it going to enforce sponsorship mode if I just provide biconomyPaymasterApiKey?

Reply:
Isn't the Paymaster set to SPONSORED by default ?